test/units/TEST-34-DYNAMICUSERMIGRATE.sh

```diff
@@ -227,6 +227,88 @@ EOF
     [[ $(stat -c "%u" /var/lib/testidmapped/testfile) == 0 ]]
 }
 
+test_quotas() {
```

Comment:
Please move this to a new test script, e.g. TEST-07-PID1.quota.sh.
Please do not reuse TEST-34.